Pipfile

```diff
@@ -0,0 +1,12 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+jellyfin-apiclient-python = {editable = true, path = "."}
+
+[dev-packages]
+
+[requires]
+python_version = "3.8"
```

Comment:
`Pipfile` is only for people who use `pipenv` and I don't think it will affect the package dependency if you don't use `pipenv` at all but I can remove it or add it in a separate commit.